### PR TITLE
Restructuring the integration/install docs

### PR DESCRIPTION
**Pull request by tylerkelly13**
Related Ticket: N/A

Description of Changes:
* Fix the "advanced installation" docs
* Break-down and reorganize the integration docs
* move custom shortcut docs into custom ui docs
* reduce duplication where possible
* increase duplication (thanks Jekyll)
* bug fixes

Pre-checks:
- [x] Branch prefixed with `feature/` or `hotfix/`
- [x] `_data/nav.yml` has been updated (if applicable)
- [x] Files has been included where required (if applicable)
- [x] Files removed have been deleted, not just excluded from the build (if applicable)
- [x] (New product features only) Release Note added

Review:
- [x] Documentation Team Lead has reviewed
- [x] Product Manager has reviewed

### CODE REVIEW

File: _includes/integrations/bootstrap-quick-start.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    <script>
+      tinymce.init({
+        selector: 'textarea#tiny'
+      });
+    </script>
+    ```
+
+## Using TinyMCE in a Bootstrap dialog
+
+To render {{site.productname}} instances inside Bootstrap UI dialogs, add the following code:
```
Comment (lnewson) [inline _includes/integrations/bootstrap-quick-start.md:51]: This definitely isn't right... the whole Bootstrap 5 section is missing here: https://www.tiny.cloud/docs/integrations/bootstrap/#usingtinymceinabootstrapdialog

File: _includes/integrations/expressjs-quick-start.md [hunk 2]
```diff
@@ -134,15 +123,15 @@ This procedure requires:
     * To start the development server, navigate to the `my_{{site.prodnamecode}}_app` directory and run:
 
         ```sh
-        $ npm start
+        npm start
         ```
 
-    * Visit http://localhost:3000 to view the application.
+    * Visit <http://localhost:3000> to view the application.
```
Comment (lnewson) [inline _includes/integrations/expressjs-quick-start.md:129]: Is this correct? As this looks like it should be a link and that isn't how links are done in Markdown.

File: _config.yml [hunk 3]
```diff
@@ -37,8 +37,10 @@ enterpriseplan: Tiny Custom Plans
 supporturl: https://support.tiny.cloud
 plugindirectory: https://www.tiny.cloud/tinymce/features/
 gettiny: https://www.tiny.cloud/get-tiny
+blogurl: https://www.tiny.cloud/blog
 download-community: https://www.tiny.cloud/get-tiny/self-hosted/
 download-enterprise: https://www.tiny.cloud/my-account/downloads/
+download-zip: https://download.tiny.cloud/tinymce/community/tinymce_latest.zip
```
Comment (lnewson) [inline _config.yml:43]: Should this be `download-community-zip`? As that'd be clearer and we may need an enterprise zip at some stage potentially :shrug: (though not overly likely atm)

Comment (tylerkelly13) [reply _config.yml:43]: Removed. after a quick search, I can't find anywhere that I've used it :thinking: I may have added it then changed the procedure?

Comment (tylerkelly13) [reply _config.yml]: fixed in https://github.com/tinymce/tinymce-docs/pull/2196/commits/9eac4d2b0b024228dc88cacbb7ef58c295ac5477

File: _includes/integrations/bootstrap-quick-start.md (hunk 1 above)
Comment (tylerkelly13) [reply _includes/integrations/bootstrap-quick-start.md:51]: fixed in https://github.com/tinymce/tinymce-docs/pull/2196/commits/9eac4d2b0b024228dc88cacbb7ef58c295ac5477

**Review by metricjs — APPROVED** [review]

**Pull request merged**